ui/chapters/src/main/java/de/danoeh/antennapod/ui/chapters/PodcastIndexTranscriptUtils.java

```diff
@@ -2,32 +2,44 @@
 
 import android.util.Log;
 
+import de.danoeh.antennapod.parser.transcript.TranscriptParser;
+import org.apache.commons.io.FileUtils;
+
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.nio.charset.Charset;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
 
 import de.danoeh.antennapod.model.feed.FeedMedia;
 import de.danoeh.antennapod.net.common.AntennapodHttpClient;
+import de.danoeh.antennapod.model.feed.Transcript;
+import okhttp3.CacheControl;
 import okhttp3.Request;
 import okhttp3.Response;
 import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
 
 public class PodcastIndexTranscriptUtils {
 
     private static final String TAG = "PodcastIndexTranscript";
 
+
```

Comment:
Please revert white space change. Might also be caused by my rebasing, not sure

Reply:
remove whitespace